source/gui/settingsDialogs.py

```diff
@@ -2546,6 +2546,27 @@ def onSave(self):
 		config.conf["documentFormatting"]["reportFrames"]=self.framesCheckBox.Value
 		config.conf["documentFormatting"]["reportClickable"]=self.clickableCheckBox.Value
 
+
+class DocumentNavigationPanel(SettingsPanel):
+	# Translators: This is the label for the document navigation settings panel.
+	title = _("Document Navigation")
+	helpId = "DocumentNavigation"
+
+	def makeSettings(self, settingsSizer: wx.BoxSizer) -> None:
+		sHelper = guiHelper.BoxSizerHelper(self, sizer=settingsSizer)
+		# Translators: This is a label for the paragraph navigation style in the document navigation dialog
+		paragraphStyleLabel = _("&Paragraph style:")
+		self.paragraphStyleCombo: nvdaControls.FeatureFlagCombo = sHelper.addLabeledControl(

```

Comment:
you should also bind a help event, see other usages of `bindHelpEvent` for similar settings